clang/lib/Driver/Driver.cpp

```diff
@@ -3250,6 +3250,9 @@ class OffloadingActionBuilder final {
     /// Type of output file for FPGA device compilation.
     types::ID FPGAOutType = types::TY_FPGA_AOCX;
 
+    /// List of objects to extract FPGA dependency info from
+    ActionList FPGAObjects;
```

Comment:
Maybe:
```suggestion
    ActionList FPGAObjectInputs;
```
to clarify that we're dealing with input actions?

Reply:
Sure - done.